tests/ut/ir/transforms/test_pass_pipeline.py

```
@@ -402,5 +402,53 @@ def test_invalid_program_throws(self):
             passes.verify_properties(props, program, "TestPass")
 
 
+class TestReportInstrument:
+    """Test ReportInstrument with report generation."""
+
+    def test_get_name(self):
+        """Default name is ReportInstrument."""
+        instrument = passes.ReportInstrument("/tmp/test_report")
```

Ruff (0.15.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/test_report"

(S108)